src/style.css

```
@@ -0,0 +1,45 @@
+.container {
+  margin: 0 ;
+  padding: 0 ;
+  box-sizing: border-box ;
+}
+
+.bod {
+  margin: 0 ;
+  padding: 0 ;
+}
+
+.text {
+  display: none ;
+}
+
+.stars__star {
+  width: 16px ;
+  height: 16px ;
+  padding-right: 4px ;
+  background: url("images/star.svg") 2px 2px /13px no-repeat ;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```

```suggestion
.stars {
```
use modificator just to set special props